niimpy/EDA/test_EDA_punchcard.py

```diff
@@ -1,113 +1,47 @@
 import pytest
+import plotly
 from niimpy.EDA import setup_dataframe
 from niimpy.EDA import EDA_punchcard
 
+def test_punchcard_one_user():
+    df = setup_dataframe.create_dataframe()
+    user_list = ['user_1']
+    columns = ['col_1']
+    title = 'test_title'
+    resample = "D"
 
-class TestEDApunchcard(object):
+    fig = EDA_punchcard.punchcard_plot(df,user_list,columns,title, resample)
+    assert (type(fig) == plotly.graph_objs._figure.Figure)
     
-    def test_EDA_punchcard(self):
-        """   
-        Returns
-        -------
-        None.
+def test_punchcard_one_user_two_columns():
+    df = setup_dataframe.create_dataframe()
+    user_list = ['user_1']
+    columns = ['col_1','col_2']
+    title = 'test_title'
+    resample = "D"
+    normalize = True
+
+    fig = EDA_punchcard.punchcard_plot(df,user_list,columns,title, resample,normalize)
+    assert (type(fig) == plotly.graph_objs._figure.Figure)
```

Comment:
The tests can be improved by checking some attributes of plotly Figure. For example, the number of xticks can be accessed by calling `fig.layout.xaxis.nticks`, title can be checked by calling `ffig.layout.title.text` etc.

Reply:
see https://plotly.com/python/figure-structure/

Reply:
Improved the test